libraries/standard/mqtt/src/iot_mqtt_serialize.c

```diff
@@ -216,6 +216,27 @@ static uint8_t * _encodeString( uint8_t * pDestination,
                                 const char * source,
                                 uint16_t sourceLength );
 
+/**
+ * @brief Get the CONNECT flags based on the given parameters.
+ * 
+ * @param[in] pConnectInfo User-provided CONNECT information.
+ * 
+ * @return CONNECT flags field for an MQTT packet.
+ */
+static uint8_t _getConnectFlags( const IotMqttConnectInfo_t * pConnectInfo );
```

Comment:
Should rename to `setConnectFlags`, this function is setting the flags in a CONNECT packet.

Reply:
I think this function could just be removed, as after testing again it seems most of the complexity from this _serializeConnect came from the username portion. Looks like leaving the connectFlags doesn't increase the complexity beyond 8.